docs/Makefile

```diff
@@ -18,3 +18,6 @@ help:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+lint:
```

Comment:
What's the expected benefit here too?

Reply:
I thought this is in the root directory, will remove this from here.

I think having a `Makefile` is beneficial to organise some scripts a bit. It can be re-used later in CI or setup local environment

I expect these will exist eventually `make install-test-requirements.txt`, `make test` ,`make lint` etc.